packages/SidePanel/src/components/Overlay/Overlay.js

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import PropTypes from "prop-types";
+import RawButton from "@paprika/raw-button";
+import { visuallyHidden } from "@paprika/stylers/lib/includes";
+import { overlayCSS } from "./Overlay.styles";
+
+const propTypes = {
+  /**
+   * Will accept the click event on outside of the sidepanel when losing focus
+   * @boolean
+   */
+  background: PropTypes.string,
+  hasOutsideClick: PropTypes.bool,
+  onClose: PropTypes.func,
+  /** Control the z position of the sidepanel overlay */
+  zIndex: PropTypes.number,
+};
+
+const defaultProps = {
+  background: "#000",
+  hasOutsideClick: true,
+  onClose: null,
+  zIndex: 99,
```

Comment:
`z(n)` where `n <= 7`.

Reply:
this doesn't work because z returns a full string `z-index: 1` I have added a new function under z named `z.toNumber(6)` which returns only the number that we need.